sklearn_numba_dpex/kmeans/kernels/compute_euclidean_distances.py

```diff
@@ -4,7 +4,9 @@
 import numpy as np
 import numba_dpex as dpex
 
-from ._base_kmeans_kernel_funcs import make_pairwise_ops_base_kernel_funcs
+from ._base_kmeans_kernel_funcs import (
+    make_pairwise_ops_base_kernel_funcs,
+)
```

Comment:
Nit: this better be reverted if nothing has changed not to obfuscate the history.

```suggestion
from ._base_kmeans_kernel_funcs import make_pairwise_ops_base_kernel_funcs
```

Reply:
this change in particular is actually nicer do you mind if I automerge it directly on main